pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -43,24 +43,40 @@ type CompactedEventList struct {
 // Used to detect the possible stack trace on logs
 var stackTraceRegex = regexp.MustCompile(`\.go:\d+\s\+0x`)
 
-// GatherClusterOperatorPodsAndEvents collects information about pods
+// GatherClusterOperatorPodsAndEvents Collects information about pods
 // and events from namespaces of degraded cluster operators. The collected
 // information includes:
 //
 // - Definitions for non-running (terminated, pending) Pods
 // - Previous (if container was terminated) and current logs of all related pod containers
 // - Namespace events
 //
-//   - Location of pod definitions: config/pod/{namespace}/{pod}.json
-//   - Location of pod container current logs:
-//     config/pod/{namespace}/logs/{pod}/{container}_current.log
-//   - Location of pod container previous logs:
-//     config/pod/{namespace}/logs/{pod}/{container}_previous.log
-//   - Location of events in archive: events/
-//   - Id in config: clusterconfig/operators_pods_and_events
-//   - Spec config for CO resources since versions:
-//   - 4.6.16+
-//   - 4.7+
+// ### API Reference
+// None
+//
+// ### Sample data
+// - docs/insights-archive-sample/pod
+// - docs/insights-archive-sample/events
+//
+// ### Location in archive
+// - `config/pod/{namespace}/{pod}.json`
+// - `events/`
+// - `config/pod/{namespace}/logs/{pod}/{container}_{current|previous}.log`
+//
+// ### Config ID
+// `clusterconfig/operators_pods_and_events`
+//
+// ### Released version
+// - 4.8.2
+//
+// ### Backported versions
+// - 4.6.35+
+// - 4.7.11+
+//
+// ### Changes
+// - This gather used to be part of [ClusterOperators](#ClusterOperators), it was split generating this gather at `4.8.2`.
```

Comment:
Nit: I am not sure I really understand the following:

> ...,it was split generating this gather at 4.8.2.

Reply:
I can rephrase it, but basically, `GatherClusterOperatorPodsAndEvents` used to be inside the `GatherClusterOperatorPodsAndEvents`, and it actually moved from it to its own gather at `4.8.2`.

Reply:
Offering a more humane wording, although a bit more verbose:

```
The data gathered by `GatherClusterOperatorPodsAndEvents` were originally gathered by [`ClusterOperators`](#ClusterOperators). The [`ClusterOperators`](#ClusterOperators) gather was split at 4.8.2 and the change was backported to 4.7.11 and 4.6.35.
```